returnn/datasets/basic.py

```diff
@@ -1424,7 +1450,10 @@ def init_dataset(
         return data
     if callable(kwargs):
         return init_dataset(
-            kwargs(), extra_kwargs=extra_kwargs, default_kwargs=default_kwargs, parent_dataset=parent_dataset
+            kwargs(),
+            extra_kwargs=extra_kwargs,
+            default_kwargs=default_kwargs,
+            parent_dataset=parent_dataset,
```

Comment:
Note: It makes it a bit unnecessary extra effort to look through the diff when there are such cosmetic changes in a bigger PR. Everytime I review again, I look through that again, only to notice that there is actually no change here. I also wonder if this looks really nicer now.

Reply:
Oh, yeah. I should probably keep the diff in mind myself. These changes happen over time. I think in this case it was because it had an additional parameter at first, forcing the breakup into multiple lines, and after the parameter was removed again, the trailing comma remained, forcing the lines still to be broken. I'll remove those.